00_Base/package.json

```diff
@@ -2,17 +2,17 @@
   "name": "@citrineos/base",
   "version": "1.0.0",
   "description": "The base module for OCPP v2.0.1 including all interfaces. This module is not intended to be used directly, but rather as a dependency for other modules.",
-  "main": "lib/index.js",
-  "types": "lib/index.d.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "files": [
-    "lib"
+    "dist"
   ],
   "scripts": {
     "prepublish": "npx eslint ./src",
-    "prepare": "npm run build",
```

Comment:
why are we removing the 'prepare' script? `prepare` always runs when you execute `npm install`. needing to execute `npm run compile` after `npm install` is a definite downgrade.

Reply:
we dont need to run compile, the `npm run build` in root does everything

Reply:
understood!